docs/guide.rst

```diff
@@ -789,11 +789,17 @@ libraries, e.g. one of the following:
   * `a native beakerlib library`__ by Dalibor Pospisil, a
     distributed version of Ondrej Moris's library, supporting any
     number of hosts.
+  * `redis server`__ by Jan Scotka, simple key value exchange
+    solution between machines. It is not direct library for synchronization,
+    but it is simple to use it also for synchronization and data transfer.
+    example in tmt repo set variables as part of preprare step for master then
```

Comment:
Let's avoid using `master` here as it is sensitive for some people.

Reply:
I don't understand the final sentence: "example in tmt repo set variables as part of preprare step for master then read the values inside test."

Reply:
Okay, we can rephrase it, but not sure how, it describes the example where you will find the setup for redis master and set values  and where are consumed these values